packages/features/bookings/Booker/components/LargeCalendar.tsx

```diff
@@ -7,19 +7,22 @@ import type { CalendarAvailableTimeslots } from "@calcom/features/calendars/week
 import { localStorage } from "@calcom/lib/webstorage";
 
 import { useBookerStore } from "../store";
-import { useEvent, useScheduleForEvent } from "../utils/event";
+import type { useScheduleForEventReturnType } from "../utils/event";
+import { useEvent } from "../utils/event";
 import { getQueryParam } from "../utils/query-param";
 import { useOverlayCalendarStore } from "./OverlayCalendar/store";
 
-export const LargeCalendar = ({ extraDays }: { extraDays: number }) => {
+export const LargeCalendar = ({
+  extraDays,
+  schedule,
+}: {
+  extraDays: number;
+  schedule: useScheduleForEventReturnType;
```

Comment:
We use "schedule.isLoading" and "schedule.data.slots" in this component - personally i would refactor prop to be schedule : {isLoading, slots} to make it cleaner instead of one accessing schedule directly other schedule.data but this is not crucial.